src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -225,6 +225,7 @@ class BtreeLBAManager : public LBAManager {
 
 
   struct alloc_mapping_info_t {
```

Comment:
Let's add factory helper method to clarify how this is supposed to be constructed, e.g.
```c++
static alloc_mapping_info_t create_zero(len);
static alloc_mapping_info_t create_indirect(len, value/* intermediate_key */));
static alloc_mapping_info_t create_direct(len, value/* paddr */, crc, extent);
```

Reply:
Done:-)